server/api.py

```diff
@@ -269,6 +293,7 @@ async def patch_vcon(vcon_uuid: UUID, plugin: str):
         await asyncio.create_task(plugin_module.run(vcon_uuid))
     except Exception as e:
         message = "Error in plugin: {} {}".format(plugin, e)
+        print(message)
```

Comment:
Do we want this print statement in the commit?

Reply:
Good catch. We can move it to the logger.